packages/admin/dashboard/src/routes/promotions/promotion-edit-details/components/edit-promotion-form/edit-promotion-details-form.tsx

```diff
@@ -58,6 +68,7 @@ export const EditPromotionDetailsForm = ({
         is_automatic: data.is_automatic === "true",
         code: data.code,
         status: data.status,
+        is_tax_inclusive: data.is_tax_inclusive,
```

Comment:
nit: should we set `is_tax_inclusive` flag to false on submit if the promotion value type is changed to `percentage`?